packages/react-ui-core/src/DropDown/DropDown.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+
+@themed(/^DropDown/, {
+  pure: true,
+})
+
+export default class DropDown extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    visible: PropTypes.bool,
+    theme: PropTypes.object,
+    children: PropTypes.node,
+    Anchor: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.func,
+    ]),
+    toggleVisibilty: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    visible: false,
+    theme: {},
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = { visible: this.props.visible }
+  }
+
+  toggleVisibilty = () => {
+    this.setState({ visible: !this.state.visible })
+  }
+
+  render() {
+    const {
+      theme,
+      visible,
+      children,
+      Anchor,
+      className,
+      ...props
+    } = this.props
+
+    const dropDownVisible = this.state.visible
+
+    return (
+      <div
+        className={classnames(
+          theme.DropDown,
+          className
+        )}
+      >
+        <Anchor
```

Comment:
How does `Anchor` work if I don't pass in the prop?

Reply:
I've add a default anchor per our discussion